### PR TITLE
Interop - Onboard MTC operator on ROSA

### PR DESCRIPTION
**Pull request by oharan2**
Depends on #45506

### CODE REVIEW

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-mtc-4.14-lp-rosa-mtc-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-images

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-mtc-interop-aws-rosa

File: ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: mtc-interop-aws-rosa
```
Comment (ascerra) [inline ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa.yaml:52]: I believe this PR is putting in place testing for Rosa Classic but I don't see anything that tells us that explicitly.

We use unique identifiers to help us separate our work

For self-managed we use "-lp-interop"
For Rosa HyperShift we us "-lp-rosa-hypershift"

For Rosa Classic what should we use?
- I'd reccomend "-lp-rosa-classic"

This unique identifier can either be in the filename or here in the test stanza name

File: ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  cron: 0 6 25 12 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      CLUSTER1_CONFIG: name=mtc-rosa-source;platform=rosa;region=us-east-2;version=4.13;channel-group=stable;compute-machine-type=m5.xlarge;timeout=2h;replicas=2
+      CLUSTER2_CONFIG: name=mtc-rosa-target;platform=rosa;region=us-east-2;version=4.14;channel-group=nightly;compute-machine-type=m5.xlarge;timeout=2h;replicas=2
+      FIREWATCH_CONFIG: |
```
[the remark this answers is not in the document]
Comment (calebevans) [reply ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa.yaml:59]: @ascerra https://github.com/openshift/release/pull/46484#pullrequestreview-1767955141

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-images

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-images

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-images

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

File: ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa-classic.yaml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: mtc-interop-aws-rosa
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      CLUSTER1_CONFIG: name=mtc-rosa-source;platform=rosa;region=us-east-2;version=4.13;channel-group=stable;compute-machine-type=m5.xlarge;timeout=2h;replicas=2
```
[the remark this answers is not in the document]
Comment (calebevans) [reply ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa-classic.yaml:65]: I can answer this - MTC's main purpose is to transfer one cluster to another cluster on a separate version. So think 4.13 -> 4.14 or 4.12 -> 4.14. The request to transfer from an n-1 cluster to a n cluster was made by PQE.

[the remark this answers is not in the document]
Comment (Simranpal) [reply ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa-classic.yaml:65]: Thanks! makes sense

**Pull request merged**